modules/springmockk/build.gradle.kts

```diff
@@ -0,0 +1,46 @@
+import buildsrc.config.Deps
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+import java.time.Duration
+
+plugins {
+    buildsrc.convention.`kotlin-jvm-spring`
+    buildsrc.convention.`mockk-publishing`
+}
+
+description = "MockBean and SpyBean, but for MockK instead of Mockito"
+
+val mavenName: String by extra("MockK springmockk")
+val mavenDescription: String by extra("${project.description}")
+
+publishing {
+    publications {
+        register<MavenPublication>("release") {
+            afterEvaluate {
+                from(components["java"])
+            }
+        }
+    }
+}
+
+tasks {
+    test {
+        useJUnitPlatform()
+        jvmArgs(
+            "--add-opens", "java.base/java.lang.reflect=ALL-UNNAMED"
+        )
+    }
+}
+
+dependencies {
+    api(projects.modules.mockk)
+    implementation(Deps.Libs.kotlinReflect)
+
+    implementation(Deps.Libs.springBootTest)
+    implementation(Deps.Libs.springTest)
+    implementation(Deps.Libs.springContext)
```

Comment:
I did further tests, dependency which break springmockk is: `org.springframework:spring-context:6.1.0` and above

Bumping Spring caused dependency mismatch. Following code works:
```
testImplementation("io.mockk:springmockk:1.13.10-SNAPSHOT") {
    exclude group: 'org.springframework.boot'
    exclude group: 'org.springframework'
}
//testImplementation 'org.springframework:spring-context:6.1.0' // this version breaks most recent version of springmockk
```

Reply:
This happens due to some internal type mismatches in `factoryBeanObjectType`. This looks like some internal interface in spring which is not standardized and changes between versions. I will try to mitigate this issue somehow.